tests/qeqiskit/backend/backend_test.py

```diff
@@ -177,36 +182,12 @@ def test_execute_with_retries_timeout(self, backend):
             for _ in range(num_jobs):
                 backend.execute_with_retries([circuit], n_samples)
 
-    def test_run_circuitset_and_measure_readout_correction_retries(
```

Comment:
Why deleting this test?

Reply:
It seemed like a dupilcate of test_readout_correction_works_run_circuitset_and_measure.

Reply:
I don't think so – this one explicitly targets "a race condition". 
But to be honest it seems to me it's broken, as the comment says that "The test will still pass".
I think better option is to leave it as it, but add `xfail` or `skip` at the beginning.